src/application/services/useNavbar.ts

```diff
@@ -71,6 +76,13 @@ export default function useNavbar(): useNavbarComposableState {
     workspaceService.patchOpenedPageByUrl(url, page);
   };
 
+  /**
+   * Delete opened pages excluding Home page
+   */
+  function deleteOpenedPages(): void {
+    workspaceService.deleteOpenPages();
```

Comment:
use same naming please
```suggestion
    workspaceService.deleteOpenedPages();
```

Reply:
Check now